### PR TITLE
Added JS for reservation form

### PR DESCRIPTION
**Pull request by zoricaUros**

### CODE REVIEW

File: assets/img/backgroundtestv1.1.jpg
Comment (Petar-Repac) [inline assets/img/backgroundtestv1.1.jpg:1]: We don't need this anymore, thankfully

File: assets/img/backgroundyellow.jpg
Comment (Petar-Repac) [inline assets/img/backgroundyellow.jpg:1]: Nor this one, into the fire with it!

File: rezervacija.html [hunk 1]
```diff
@@ -335,6 +385,115 @@ <h3>Aero parking</h3>
...
+          return;
+        }
+       
+        // number of days
+        let output = Math.ceil(((secondDate - firstDate) / (1000 * 60 * 60 * 24)));
+
+        //  arrival after 22h and departure before 2AM
+        let arrivalHour = firstDate.getHours();
+        let departureHour = secondDate.getHours();
+        if (arrivalHour >= 22 || departureHour < 2) {
```
Comment (Petar-Repac) [inline rezervacija.html]: 😎🤏  🤨🕶️🤏

File: rezervacija.html [hunk 2]
```diff
@@ -335,6 +385,115 @@ <h3>Aero parking</h3>
...
+        let secondDate = new Date(departureDate);
+
+        // invalid input
+        if (secondDate < firstDate) {
+          formCharge.textContent = "Datum odlaska mora biti posle polaska, molimo unesite ispravan datum!";
+          return;
+        }
+       
+        // number of days
+        let output = Math.ceil(((secondDate - firstDate) / (1000 * 60 * 60 * 24)));
```
Comment (Petar-Repac) [inline rezervacija.html]: Your comment should've been the variable name, and there would be no need for the comment then.

File: rezervacija.html [hunk 3]
```diff
@@ -335,6 +385,115 @@ <h3>Aero parking</h3>
...
+
+        //  arrival after 22h and departure before 2AM
+        let arrivalHour = firstDate.getHours();
+        let departureHour = secondDate.getHours();
+        if (arrivalHour >= 22 || departureHour < 2) {
+          output = output - 1;
+          return;
+        }
+
+        // more then 40 days
```
Comment (Petar-Repac) [inline rezervacija.html]: than*

File: assets/img/backgroundtestv1.1.jpg
Comment (zoricaUros) [reply assets/img/backgroundtestv1.1.jpg:1]: ok, do you want me to remove it?

**Pull request merged**